edvsystem/wasch.cgi

```diff
@@ -671,19 +671,26 @@ sub print_header {
           my @file = <START>;
           print @file;
      close (START);
+     print "<head>";
```

Comment:
The right place to add this might have been [inc/start.inc](https://github.com/pizzard/waschagtvk/blob/master/edvsystem/inc/start.inc), but that file seems to be out of date and some needed files such as `formate.css` are missing in the repository.